clang/lib/CIR/CodeGen/CIRGenItaniumCXXABI.cpp

```diff
@@ -945,13 +944,9 @@ CIRGenCallee CIRGenItaniumCXXABI::getVirtualFunctionPointer(
     if (CGM.getItaniumVTableContext().isRelativeLayout()) {
       llvm_unreachable("NYI");
     } else {
-      VTable = CGF.getBuilder().createBitcast(
-          loc, VTable, CGF.getBuilder().getPointerTo(TyPtr));
-      auto VTableSlotPtr = CGF.getBuilder().create<cir::VTableAddrPointOp>(
-          loc, CGF.getBuilder().getPointerTo(TyPtr),
-          ::mlir::FlatSymbolRefAttr{}, VTable,
-          cir::AddressPointAttr::get(CGF.getBuilder().getContext(), 0,
-                                     VTableIndex));
+      auto VTableSlotPtr =
+          CGF.getBuilder().create<cir::VTableGetVirtualFnAddrOp>(
```

Comment:
nit:  add `auto &builder = CGM.getBuilder();` at top of the function to clean up multiple uses of the builder